.travis.yml

```diff
@@ -33,6 +35,10 @@ matrix:
     - env:
         - JOB="3.7" ENV_FILE="ci/deps/travis-37.yaml" PATTERN="(not slow and not network and not clipboard)"
 
+    - arch: arm64
+      env:
+        - JOB="3.7" PYTEST_WORKERS=8 ENV_FILE="ci/deps/travis-37-arm64.yaml" PATTERN="(not slow and not network and not clipboard)"
```

Comment:
you are likely to have much better performance with 2 workers here. I would guess that's what we have physically, and what we use on other processors.

Reply:
I have updated to 2 workers as suggested. Please review.

Reply:
it still using 8 on the arm build - make this 2

and taking 56 minutes which is too long 

put the build in the allowed failures section

Reply:
I have updated to 2 workers in the previous commit, but the build was getting stalled and failing while running tests. So, I have moved back to 8 workers. Pandas compilation and test execution are taking more time to complete in ARM64 when compared to the AMD64 platform.